### DISCUSSION TITLE
Update impact_data.py to work with new version of EMDAT

### DISCUSSION THREAD

**Discussion by luseverin** [CLIMADA-project/climada_python#883] [General]
Impact data downloaded from EMDAT now come in an updated (2024) format (see https://doc.emdat.be/docs/data-structure-and-content/emdat-public-table/). As some of the columns headers were changed and columns were added/removed, it seems to me that the `impact_data.py` should be updated in order to work properly with the new version of the data. I think this requires updating the `VARNAMES_EMDAT` dictionary, and then testing that the rest of the functions in the module work. I have created a new branch (`feature/update_emdat_impact_data`) and have started updating the`VARNAMES_EMDAT` dictionary but have not checked yet if the rest of the functions in the module were working. 
I would suggest that someone who is used to work with EMDAT data and knows well the `impact_data.py` have a look and let me know what they think.

**Comment by chahank** [comment]
This is a good point. One question I had using the module is how we should continue using it. It looks like this module was originally built for one specific use case with tropical cyclones. Would a refactoring of it to be more generic and easier to update the data formats in the future be worth it? I also have written a few methods in a separate project to match hazard events to emdat recorded values which are a bit different from the ones in the module. Probably others have also done so. 

I think we can convert this issue to a discussion.
